cordapp/src/main/kotlin/net/corda/plugins/Info.kt

```diff
@@ -4,11 +4,21 @@ import org.gradle.api.tasks.Input
 
 open class Info {
     @get:Input
-    var name: String? = null
+    var cordappContractName: String? = null
     @get:Input
-    var version: String? = null
+    var cordappContractVersion: String? = null
```

Comment:
Maybe `Int` ?

Reply:
@szymonsztuka lets keep it as a String in the gradle plugins to cater for future evolvability.
But we will treat it as an integer for the purposes of Contract version downgrade checking.